sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2666,6 +2666,19 @@ def __setstate__(self, state):
         if version < 6:
             state['cluster_name_on_cloud'] = state['cluster_name']
 
+        if version < 7:
+            launched_resources = state['launched_resources']
+
+            # Backward compatibility: we change the default value for TPU VM to
```

Comment:
QQ: why handle here vs. in https://github.com/skypilot-org/skypilot/blob/88562fe67aa11db771b546e24a7fd04015309e6d/sky/resources.py#L47?

Reply:
Good catch! Moved it to resources instead. : )